setup/setup_locations_template.sh

```diff
@@ -0,0 +1,2 @@
+echo 'Copy this file to setup/setup_locations.sh and modify the directory to point to the directory of your OpSim output'
```

Comment:
If we make the opsim db file a required argument of the `generatePhosimInput.py` script, this setup file, with its current contents, would not be needed.